pkg/trace/info/stats_test.go

```diff
@@ -0,0 +1,47 @@
+package info
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTracesDropped(t *testing.T) {
+	var s TracesDropped
+	s.DecodingError++
+	s.ForeignSpan++
+
+	t.Run("StatsToMap", func(t *testing.T) {
+		for k, v := range s.tagValues() {
+			if k == "decoding_error" || k == "foreign_span" {
+				assert.EqualValues(t, v, 1)
+			} else {
+				assert.EqualValues(t, v, 0)
+			}
+		}
+	})
+
+	t.Run("StatsToString", func(t *testing.T) {
+		assert.Equal(t, "decoding_error:1, foreign_span:1", s.String())
```

Comment:
This test is weird. You could've used a [for loop with multiple test cases](https://github.com/golang/go/wiki/TableDrivenTests), having the input be a `TracesDropped` structure, asserting against a string output. This way, you're only testing two keys out of all.

Reply:
It's just a sanity check that the function works, and that the keys are sorted correctly. We could enumerate all the test cases but that's probably overkill. I added a couple more keys, but I don't think enumerating lots of possible combinations of keys here is going to make the test any more useful.